tools/exports.js

```diff
@@ -1,15 +1,4 @@
-exports["Compressor"] = Compressor;
 exports["Dictionary"] = Dictionary;
-exports["JS_Parse_Error"] = JS_Parse_Error;
```

Comment:
Would I be a heretic to suggest a new `minify()` option `{ catch: true }` and emit the exception to `result.errors` or `result.exception` or equivalent?